woocommerce/woocommerce-android
# [Woo POS] Changes to more screen to accomodate entry point to POS

Closes: #11545


### Description

The PR adds changes to implement an updated UI for the "more" menu screen and adds POS button to it.

As requested by Joe also added "external link" icons for Wp-admin and view store.

The red badge is not implemented here.

<img width="865" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/d80c571a-e9e3-4801-8905-922922d0dfd8">


### Testing instructions

* Go to the More screen and make sure there is no changes apart from the UI. Check dark mode. Make sure that all the buttons are in place as they used to be except the POS is on top now

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/cc39ac85-15f7-4a23-9d59-112e872f67e4

<img width="250" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/b28955ac-fe36-49bc-b55f-be0bf429ca27">
